src/components/CurrencyInputPanel/CurrencyInputPanel.component.tsx

```diff
@@ -74,6 +76,28 @@ export const CurrencyInputPanelComponent = ({
     }
   }, [disableCurrencySelect, isLoading])
 
+  const handleOnMax = useCallback(() => {
+    if (onMax && setIsMaxAmount) {
```

Comment:
There is no need to check if `setIsMaxAmount` exists because you declared it above.

Reply:
Addressed in [60ad9c7](https://github.com/levelkdev/swapr-dapp/pull/1013/commits/60ad9c7e9feff96ac053e0555e59897acb3af6fd). Thanks!